### PR TITLE
dialog_actions_on_end_task

### PR DESCRIPTION
**Pull request by shayb**
add action after task is complete and user click finish after got reward

### CODE REVIEW

File: projectFilesBackup1/.idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,4287 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (berryve) [inline projectFilesBackup1/.idea/workspace.xml]: as you said - please remove this file and instead add an entry in .gitignore to ignore it.

File: app/src/main/java/org/kinecosystem/kinit/model/earn/Task.kt [hunk 2]
```diff
@@ -26,6 +26,8 @@ data class Task(
     val minToComplete: Float? = null,
     @SerializedName("tags")
     val tags: List<String>? = null,
+    @SerializedName("post_task_actions")
+    val postTaskActions: List<PostTaskAction>,
```
Comment (berryve) [inline app/src/main/java/org/kinecosystem/kinit/model/earn/Task.kt]: 1. need to change this to List<PostTaskAction> ?

File: app/src/main/java/org/kinecosystem/kinit/util/BindingUtils.java [hunk 3]
```diff
@@ -160,4 +161,11 @@ public static void setVisibilityOn(EditText view, boolean visible) {
             view.setText("");
         }
     }
+
+    @BindingAdapter({"complete"})
+    public static void updateComplete(TransactionLayoutView layoutView, boolean transactionComplete) {
+        if (transactionComplete) {
+            layoutView.updateComplete();
```
Comment (berryve) [inline app/src/main/java/org/kinecosystem/kinit/util/BindingUtils.java]: I think this should either stay inside TransactionLayoutView or changed into a more generic method where you call a generic action method depending on a specific boolean value.

Comment (shayb) [reply app/src/main/java/org/kinecosystem/kinit/util/BindingUtils.java]: that was part of transactionlayout view but i moved it to here cause all binding methods are in single place

Comment (berryve) [reply app/src/main/java/org/kinecosystem/kinit/util/BindingUtils.java]: In the same way you don't want to create a GenericUtils file that contains all private methods that a specific class wants to use - you don't want to add this specific binding method to BindingUtils. 
Either make a generic utility method or move it back to the specific class it serves.

**Pull request merged**